src/antelope/stores/evm.ts

```diff
@@ -80,6 +81,11 @@ export const useEVMStore = defineStore(store_name, {
         trace: createTraceFunction(store_name),
         init: () => {
 
+            // bypass provider detection if using WalletConnect
+            if (localStorage.getItem('wagmi.connected')){
+                return;
+            }
```

Comment:
this line prevents` evm.setSupportsMetaMask(true);` from happening, and the home page is asking const `supportsMetamask = computed(() => useEVMStore().isMetamaskSupported` which returns false and that generates the button to show the "Install Metamask" label

Reply:
Changed this to only return on mobile which resolves the issue.